### PR TITLE
Update crypto.md

### PR DESCRIPTION
**Pull request by james-chf**
Removed some TODOs/links to the closed issue https://github.com/anoma/anoma/issues/225, unsure if that still applies.

### CODE REVIEW

File: docs/src/specs/crypto.md [hunk 1]
```diff
@@ -5,12 +5,10 @@ Anoma currently supports Ed25519 signatures with more to be added:
 - [`Secp256k1`](https://github.com/anoma/anoma/issues/162)
 - [`Sr25519`](https://github.com/anoma/anoma/issues/646)
 
-Please note that the Anoma's crypto public API and encoding is currently undergoing some breaking changes with <https://github.com/anoma/anoma/issues/225>.
-
 ## Public keys
 
-A public key is a [Borsh encoded `PublicKey`](encoding.md#publickey). For the Ed25519 scheme, this is 32 bytes of Ed25519 public key, prefixed with `32` in little endian encoding (`[32, 0, 0, 0]` in raw bytes or `20000000` in hex). (TODO this will change with <https://github.com/anoma/anoma/issues/225>)
+A public key is a [Borsh encoded `PublicKey`](encoding.md#publickey). For the Ed25519 scheme, this is 32 bytes of Ed25519 public key, prefixed with `32` in little endian encoding (`[32, 0, 0, 0]` in raw bytes or `20000000` in hex).
```
Comment (james-chf) [inline docs/src/specs/crypto.md:10]: I'm unsure of the context here for where we're using Borsh-encoded public keys and signatures, but I'm guessing it's when we store them in blockchain state?

**Comment by james-chf** [comment]
Re "Anoma currently supports Ed25519 signatures" - I'm unsure here of the context where we are supporting the signatures, but I'm guessing for (at least) signing transactions and possibly anywhere else in Anoma that signatures are done?

File: docs/src/specs/crypto.md (hunk 1 above)
Comment (james-chf) [reply docs/src/specs/crypto.md:10]: I'd be tempted to remove references to encoding here, that could be covered in the next section (https://dev.anoma.net/master/specs/encoding.html)

Comment (tzemanovic) [reply docs/src/specs/crypto.md:10]: yeah, I think this is outdated now. We're using the `key::common` schema with enum cases for ed25519 and upcoming secp256k1

**Review by tzemanovic — APPROVED** [review]
LGTM, thanks! I think in future we'll add more details here about the common signing interface

**Comment by brentstone** [comment]
I suppose when the branch brent/162/secp256k1-support is ready for merge, I will update this as well

**Comment by tzemanovic** [comment]
moved to https://github.com/anoma/namada/pull/288

**Pull request closed without merging**